hack/generate-manifest.sh

```diff
@@ -314,6 +320,12 @@ if $FLEXIBLE_IPAM; then
     sed -i.bak -E "s/^[[:space:]]*#[[:space:]]*noSNAT[[:space:]]*:[[:space:]]*[a-z]+[[:space:]]*$/noSNAT: true/" antrea-agent.conf
 fi
 
+if $MULTICAST; then
+    sed -i.bak -E "s/^[[:space:]]*#[[:space:]]*trafficEncapMode[[:space:]]*:[[:space:]]*[a-z]+[[:space:]]*$/trafficEncapMode: noEncap/" antrea-agent.conf
+    sed -i.bak -E "s/^[[:space:]]*#[[:space:]]*Multicast[[:space:]]*:[[:space:]]*[a-z]+[[:space:]]*$/  Multicast: true/" antrea-agent.conf
+    sed -i.bak -E "s/^[[:space:]]*#[[:space:]]*multicastInterfaces[[:space:]]*:[[:space:]]*\[([a-zA-Z0-9]*,[[:space:]]*)*[a-zA-Z0-9]*\][[:space:]]*$/multicastInterfaces: [ens224]/" antrea-agent.conf
```

Comment:
Is this for testing multicast traffic on multiple interfaces? or ens224 is the transport interface?

Reply:
ens224 is the extra interface added for testing multicast traffic on multiple interfaces. In this testbed, ens192 is the transport interface.

Reply:
If this is specific to the testbed, why is it hardcoded in this script? This script is used to produce manifests in a generic way, it is not tied to our CI.
The interface name could be passed as a parameter maybe? This way the interface name is only hardcoded in test.sh.